### PR TITLE
[SeaTunnel]Simply seatunnel package pipeline.

### PR DESCRIPTION
**Pull request by hk-lrzy**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Comment by CalvinKirs** [comment]
hi, could you add the doc for this? https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/setup.md

**Comment by hk-lrzy** [comment]
> hi, could you add the doc for this? https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/setup.md

Sure, i will add it in next commit.

**Comment by hk-lrzy** [comment]
> > hi, could you add the doc for this? https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/setup.md
> 
> Sure, i will add it in next commit.

I already modify the setup docs, plz take a look again, thanks.

File: seatunnel-dist/pom.xml [hunk 1]
```diff
@@ -57,6 +57,37 @@
...
+                    <artifactId>seatunnel-flink-starter</artifactId>
+                    <version>${project.version}</version>
+                </dependency>
+
+                <dependency>
+                    <groupId>org.apache.seatunnel</groupId>
+                    <artifactId>seatunnel-spark-starter</artifactId>
+                    <version>${project.version}</version>
+                </dependency>
             </dependencies>
```
Comment (CalvinKirs) [inline seatunnel-dist/pom.xml:91]: Have you tested this, I think you are missing `seatunnel-connectors-v2`

Comment (hk-lrzy) [reply seatunnel-dist/pom.xml:91]: i tested it, `seatunnel-connectors-v2` already contans in `seatunnel-connectors-v2-dist`

Comment (hk-lrzy) [reply seatunnel-dist/pom.xml:91]: 截屏2022-08-30 17 48 50

Comment (CalvinKirs) [reply seatunnel-dist/pom.xml:91]: Ah,i ignored this

**Review by CalvinKirs — APPROVED** [review]

**Comment by EricJoy2048** [comment]
+1

**Pull request merged**